tokens/color/status.yaml

```diff
@@ -0,0 +1,48 @@
+color:
```

Comment:
@bennypowers, do we need a set of status tokens at this level, or can we add status tokens at the element token level only? 

For example, we want red-orange to mean danger. However, the shade of red-orange that we would want to use for a danger button background color and a danger alert background color might be different. This makes creating an `rh-status-danger-background` token difficult.